### ISSUE TITLE
hide_tabs_with_one_tab.css no longer works in FF 106

### ISSUE THREAD

**Issue by steve3386** [MrOtherGuy/firefox-csshacks#216]
hide_tabs_with_one_tab.css no longer works in FF 106.

I think it broke because they added a "tab list" button on the right side of the tab bar.

**Comment by MrOtherGuy** [comment]
Alltabs-button does affect the style yes, [see here for some more info](https://github.com/MrOtherGuy/firefox-csshacks/issues/207#issuecomment-1257511354)

However, there's other changes in the pipeline that are going to affect that style in 107 and 108. It's currently unclear to me if that style can even work at all starting from 107.

**Comment by glennpc** [comment]
You can hide the "List All Tabs" button by doing the following:

Go into about:config and set the browser.tabs.tabmanager.enabled option to false.

**Comment by steve3386** [comment]
That worked.  Thanks!  
In a message dated 10/20/2022 8:13:09 PM Eastern Standard Time, ***@***.*** writes: 

 

You can hide the "List All Tabs" button by doing the following:

Go into about:config and set the browser.tabs.tabmanager.enabled option to false.

**Comment by MrOtherGuy** [comment]
Yeah, so while removing alltabs-button and firefox-view button in some way does work in Firefox 106, I think I have to deprecate hide_tabs_with_one_tab.css from 107 onwards.

However, you can use [hide_tabs_with_one_tab_w_window_controls.css](https://github.com/MrOtherGuy/firefox-csshacks/blob/master/chrome/hide_tabs_with_one_tab_w_window_controls.css) which at the time of writing does work (with my new patches) in current nightly 108.

**Comment by 2xsaiko** [comment]
hide_tabs_with_one_tab.css with browser.tabs.tabmanager.enabled=false does completely hide the tab bar for me, while hide_tabs_with_one_tab_w_window_controls.css leaves a 20px high empty bar (same as in #207?) in place of the tab bar when there's only one tab, regardless of tabmanager setting. I couldn't figure out what causes it. This is on macOS with Firefox 107.
